app/views/shared/_card-service.html.erb

```diff
@@ -1,11 +1,13 @@
-<div class="card-service">
-  <img src="https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/greece.jpg">
-  <div class="card-service-infos">
-    <div>
-      <h2><%= service.name %></h2>
-      <p><%= service.content %></p>
+<% @services.each do |service| %>
```

Comment:
All occurrences of **@service** parameter needs to be replaced with **service** as this is the parameter sent to the partial when it is called.
<%= render 'shared/service_form', **service**: Service.new %>